src/item-user-lists.ts

```diff
@@ -151,10 +156,18 @@ export class ItemUserLists extends LitElement {
     await this.userListsService?.addMemberToList(listId, {
       identifier: this.itemId,
     });
+    this.dispatchEvent(
+      new CustomEvent('addMember', { detail: { listId, itemId: this.itemId } })
+    );
   }
 
   private async removeMember(listId: string, memberId: string): Promise<void> {
     await this.userListsService?.removeMemberFromList(listId, memberId);
+    this.dispatchEvent(
+      new CustomEvent('removeMember', {
+        detail: { listId, itemId: this.itemId },
+      })
+    );
   }
```

Comment:
Q: do we want these to be named `memberAdded` and `memberRemoved` for consistency with the above?

Reply:
keeping it to match function name and action for the events calls - to keep close to this code.
when received in petabox, it will change to `memberAdded` as that is what we want in analytics.